src/PayU/Alu/Client.php

```diff
@@ -71,101 +45,12 @@ public function setCustomUrl($fullUrl)
-            $response->setAmount((string)$xmlObject->AMOUNT);
-        }
-        if (property_exists($xmlObject, 'CURRENCY')) {
-            $response->setCurrency((string)$xmlObject->CURRENCY);
-        }
-        if (property_exists($xmlObject, 'INSTALLMENTS_NO')) {
-            $response->setInstallmentsNo((string)$xmlObject->INSTALLMENTS_NO);
-        }
-        if (property_exists($xmlObject, 'CARD_PROGRAM_NAME')) {
-            $response->setCardProgramName((string)$xmlObject->CARD_PROGRAM_NAME);
-        }
-
-        // parameters used on ALU v2 and v3
-        if (property_exists($xmlObject, 'ORDER_REF')) {
-            $response->setOrderRef((string) $xmlObject->ORDER_REF);
-        }
-        if (property_exists($xmlObject, 'AUTH_CODE')) {
-            $response->setAuthCode((string)$xmlObject->AUTH_CODE);
-        }
-        if (property_exists($xmlObject, 'RRN')) {
-            $response->setRrn((string)$xmlObject->RRN);
-        }
-
-        if (property_exists($xmlObject, 'URL_REDIRECT')) {
-            $response->setUrlRedirect((string)$xmlObject->URL_REDIRECT);
-        }
-
-        $response->parseAdditionalParameters($xmlObject);
-
-        if (property_exists($xmlObject, 'TOKEN_HASH')) {
-            $response->setTokenHash((string)$xmlObject->TOKEN_HASH);
-        }
-
-        // parameters used for wire payments on ALU v3
-        if (property_exists($xmlObject, 'WIRE_ACCOUNTS') && count($xmlObject->WIRE_ACCOUNTS->ITEM) > 0) {
-            foreach ($xmlObject->WIRE_ACCOUNTS->ITEM as $account) {
-                $response->addWireAccount($this->getResponseWireAccount($account));
-            }
-        }
-
-        return $response;
-    }
-
-    /**
-     * @param SimpleXMLElement $account
-     * @return ResponseWireAccount
-     */
-    private function getResponseWireAccount(SimpleXMLElement $account)
-    {
-        $responseWireAccount = new ResponseWireAccount();
-        $responseWireAccount->setBankIdentifier((string)$account->BANK_IDENTIFIER);
-        $responseWireAccount->setBankAccount((string)$account->BANK_ACCOUNT);
-        $responseWireAccount->setRoutingNumber((string)$account->ROUTING_NUMBER);
-        $responseWireAccount->setIbanAccount((string)$account->IBAN_ACCOUNT);
-        $responseWireAccount->setBankSwift((string)$account->BANK_SWIFT);
-        $responseWireAccount->setCountry((string)$account->COUNTRY);
-        $responseWireAccount->setWireRecipientName((string)$account->WIRE_RECIPIENT_NAME);
-        $responseWireAccount->setWireRecipientVatId((string)$account->WIRE_RECIPIENT_VAT_ID);
-
-        return $responseWireAccount;
-    }
-
     /**
      * @param Request $request
      * @param HTTPClient $httpClient
      * @param HashService $hashService
-     * @throws ClientException
-     * @throws ConnectionException
      * @return Response
+     * @throws ClientException
      */
     public function pay(Request $request, HTTPClient $httpClient = null, HashService $hashService = null)
     {
```

Comment:
If we marked HashService and HTTPClient as deprecated maybe we should stop using them as well and replace them with their V3 counterparts